src/extension/src/TelemetryWriter.py

```diff
@@ -66,7 +66,8 @@ def __ensure_message_restriction_compliance(self, full_message):
     def write_event(self, message, event_level=Constants.TelemetryEventLevel.Informational, task_name=Constants.TELEMETRY_TASK_NAME):
         """ Creates and writes event to event file after validating none of the telemetry size restrictions are breached """
         try:
-            if self.events_folder_path is None or not os.path.exists(self.events_folder_path) or not Constants.TELEMETRY_ENABLED_AT_EXTENSION:
+
+            if self.events_folder_path is None or not Constants.TELEMETRY_ENABLED_AT_EXTENSION:
```

Comment:
Keep the events_folder exists check as is.

Reply:
Will do.